Pylint: W0223

```bash
mmpose/models/detectors/bottom_up.py:19:0: W0223: Method 'simple_test' is abstract in class 'BasePose' but is not overridden (abstract-method)
mmpose/models/detectors/top_down.py:18:0: W0223: Method 'simple_test' is abstract in class 'BasePose' but is not overridden (abstract-method)
```

--- a/mmpose/models/detectors/base.py
+++ b/mmpose/models/detectors/base.py
@@ -32,10 +32,6 @@ class BasePose(nn.Module):
         """Defines the computation performed at testing."""
 
     @abstractmethod
-    def simple_test(self, img, img_metas, **kwargs):
-        """Simple test function."""
-
-    @abstractmethod
     def forward(self, img, img_metas, return_loss=True, **kwargs):
         """Forward function."""
 

